src/components/FormLayout/__tests__/FormLayout.spec.tsx

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import {
+  expect,
+  test,
+} from '@playwright/experimental-ct-react';
+import { propTests } from '../../../../tests/playwright';
+import {
+  FormLayoutForTest,
+  FormLayoutWithCustomFieldForTest,
+  FormLayoutWithoutChildrenForTest,
+} from './FormLayout.story';
+import { fieldWidthPropTest } from './_propTests/fieldWidthPropTest';
+import { fieldLayoutPropTest } from './_propTests/fieldLayoutPropTest';
+import { autoWidthPropTest } from './_propTests/autoWidthPropTest';
+import { customFieldLabelAlignmentPropTest } from './_propTests/customFieldTests/customFieldLabelAlignmentPropTest';
+import { customFieldValidationStatesPropTest } from './_propTests/customFieldTests/customFieldValidationStatesPropTest';
+import { customFieldInnerFieldSizePropTest } from './_propTests/customFieldTests/customFieldInnerFieldSizePropTest';
+import { customFieldRequiredPropTest } from './_propTests/customFieldTests/customFieldRequiredPropTest';
+
+test.describe('FormLayout', () => {
+  test.describe('base', () => {
+    test.describe('visual', () => {
+      [
+        ...propTests.defaultComponentPropTest,
+        ...fieldWidthPropTest,
```

Comment:
This is incorrect prop test name. `fieldWidthPropTest` -> `labelWidthPropTest`. Please, order those 3 alphabetically as agreed earlier.